homeassistant/components/otp/strings.json

```diff
@@ -4,13 +4,22 @@
       "user": {
         "data": {
           "name": "[%key:common::config_flow::data::name%]",
-          "token": "Authenticator token (OTP)"
+          "token": "Authenticator token (OTP)",
+          "new_token": "Generate a new token?"
+        }
+      },
+      "confirm": {
+        "title": "Verify One-Time Password (OTP)",
+        "description": "Before completing setup of One-Time Password (OTP), confirm with a verification code. Scan the QR code with your authentication app. If you don't have one, we recommend either [Google Authenticator](https://support.google.com/accounts/answer/1066447) or [Authy](https://authy.com/).\n\nAfter scanning the code, enter the six digit code from your app to verify the setup. If you have problems scanning the QR code, do a manual setup with code **`{code}`**.",
```

Comment:
URLs used in strings, need to be added using translations placeholders from code. This is for two reasons:

1. They won't accidentally become translated (and turned into 404s)
2. If an URL changes, it can be done without invalidating all translations.

Reply:
Will do. Maybe you wanna change it here too, i copied the text and adjusted it a bit:
https://github.com/home-assistant/core/blob/0b5c533669d8138344dbe9bac6f9be330307c03f/homeassistant/components/auth/strings.json#L8

Reply:
Moved also App names to placeholders, just in case the recommendations have to be changed. Not because of Authy but because Google is known for renaming constantly their products or discontinuing them 😂